sdk/metric/example_test.go

```diff
@@ -58,3 +61,134 @@ func Example() {
 	// See the go.opentelemetry.io/otel/metric package for more information
 	// about the metric API.
 }
```

Comment:
This `Example()` is the same as the one you added in `sdk/metric/example/example_test.go`. Do we need both?

Reply:
I would keep runnable examples in a location where they can be rendered in the package documentation.